pkg/collector/corechecks/ebpf/probe/tcp_queue_length.go

```diff
@@ -1,15 +1,24 @@
-// +build linux_bpf,bcc
+// +build linux_bpf
+
+//go:generate go run ../../../../ebpf/include_headers.go ../c/tcp-queue-length-kern.c ../../../../ebpf/bytecode/build/runtime/tcp-queue-length.c ../../../../ebpf/c
+//go:generate go run ../../../../ebpf/bytecode/runtime/integrity.go ../../../../ebpf/bytecode/build/runtime/tcp-queue-length.c ../../../../ebpf/bytecode/runtime/tcp-queue-length.go runtime
 
 package probe
 
 import (
 	"fmt"
+	"math"
 	"unsafe"
 
+	bpflib "github.com/DataDog/ebpf"
+	"github.com/DataDog/ebpf/manager"
+	"golang.org/x/sys/unix"
+
 	"github.com/DataDog/datadog-agent/pkg/ebpf"
+	"github.com/DataDog/datadog-agent/pkg/ebpf/bytecode/runtime"
 	"github.com/DataDog/datadog-agent/pkg/util/log"
+	"github.com/pkg/errors"
```

Comment:
I think this can be the stdlib `errors` package.

Reply:
the `Wrap` seems to be only defined in `github.com/pkg/errors`

Reply:
That is because you are supposed to do `fmt.Errorf("%w", err)` to wrap with the stdlib.